### PR TITLE
Added definition for new parameter user_splits for R.  

### PR DESCRIPTION
**Pull request by wendycwong**
This is causing cran errors according to Michal Raska.  Thank you.

### CODE REVIEW

File: h2o-r/h2o-package/R/models.R [hunk 1]
```diff
@@ -3043,11 +3043,15 @@ h2o.cross_validation_predictions <- function(object) {
 #' @param data An H2OFrame object used for scoring and constructing the plot.
 #' @param cols Feature(s) for which partial dependence will be calculated.
 #' @param destination_key An key reference to the created partial dependence tables in H2O.
-#' @param nbins Number of bins used. For categorical columns make sure the number of bins exceed the level count.  If you enable add_missing_NA, the returned length will be nbin+1.
+#' @param nbins Number of bins used. For categorical columns make sure the number of bins exceed the level count.
```
Comment (angela0xdata) [inline h2o-r/h2o-package/R/models.R]: Existing grammar issue. Change "exceed" to "exceeds" in this.
"... the number of bins exceeds the level count."

File: h2o-r/h2o-package/R/models.R [hunk 2]
```diff
@@ -3043,11 +3043,15 @@ h2o.cross_validation_predictions <- function(object) {
...
+#'        If you enable add_missing_NA, the returned length will be nbin+1.
 #' @param plot A logical specifying whether to plot partial dependence table.
 #' @param plot_stddev A logical specifying whether to add std err to partial dependence plot.
 #' @param weight_column A string denoting which column of data should be used as the weight column.
 #' @param include_na A logical specifying whether missing value should be included in the Feature values.
+#' @param user_splits A two level nested lists containing user defined split-points for pdp plots for each column.
+#'        If there are two columns using user defined split points, there should be two lists in the nested list.
+#'        Inside each list, the first element is the column name followed by values defined by user.
```
Comment (angela0xdata) [inline h2o-r/h2o-package/R/models.R]: Change to:
#' @param user_splits A two-level nested list containing user defined split points for pdp plots for each column.
#'        If there are two columns using user defined split points, there should be two lists in the nested list.
#'        Inside each list, the first element is the column name followed by values defined by the user.

File: h2o-r/h2o-package/R/models.R (hunk 1 above)
Comment (wendycwong) [reply h2o-r/h2o-package/R/models.R]: Done

File: h2o-r/h2o-package/R/models.R (hunk 2 above)
Comment (wendycwong) [reply h2o-r/h2o-package/R/models.R]: Done

**Pull request merged**